alchemy/src/cloudflare/bindings.ts

```diff
@@ -34,7 +35,8 @@ export type Binding =
   | string
   | VectorizeIndex
   | Worker
-  | Workflow;
+  | Workflow
+  | AiGateway;
```

Comment:
You also need to modify `bound.ts` which does the mapping of our `AiGateway` resource to cloudflare's `AiGateway` runtime binding type.